### ISSUE TITLE
Wine - White Grapes and Red Grapes

### ISSUE THREAD

**Issue by DianeKEPR** [FoodOntology/foodon#145]
A grape plant produces fruit berries known as grapes.  The berry is a subpart of the organism and the berry has several organism parts:  the pulp, seeds, and skins.  The color of the skins has main classifications as Red Grapes and White Grapes.  Red Grape Skin applies to a grape with a range of colors from red, to deep red, to a dark almost a blackish purple grape skin color.  White grapes skin resulted from a genetic mutation that does not produce a red pigment, and thus has a mixture of colors green and yellow to result in various hues. This all technically applies at the grape plant level, as it applies to the various species of grape plants. 
The ontology can be modeled to fit the description above, but it might be overly complicated to query for the purpose of describing a fit for purpose wine style (Red Wine, White Wine, Rose Wine), dried grape (raisin style) etc.
**Proposed:**  Create a subclass under each grape species to group cultivars of Grape Plants.  This could be like Vitis Vinifera, having two subclasses of Vitis Vinifera White Grape Plant, and Vitis Vinifera Red Grape Plant.  The various grape plant cultivars would be subclasses grouped under either White Grape Plant or Red Grape Plant.  
White grape plant cultivars would include: Chardonnay Grape Plant, White Tempranillo Grape Plant, Riesling Grape Plant, Semillon Grape Plant, Viongier Grape Plant.   Red grape plant cultivars include:  Cabernet Sauvignon grape plant, Carmenere grape plant, Grenache grape plant etc.
Each could have a synonym short name dropping the grape plant portion of the term to result in terms:  Chardonnay, White Tempranillo, Riesling, Semillon, Viongier, etc. for White grape plants.  A synonym short name for red grape plants could thus be:  Cabernet Sauvignon, Pinot Noir, Tempranillo, Malbec, Merlot, Granache, Mourvedre, Zinfandel, etc.
The Norton grape plant is a red grape plant that belongs to the species Vita aestivalis.

**Comment by DianeKEPR** [comment]
Proposal 2:  Since grape wine can be produced from more than one species of grape plant, thinking it might be better to attribute the grape skin color for White Grapes and Red Grapes at the cultivar level, not as subClasses under each of the approximately 5 species of grape plants that are in the ontology.

**Comment by maweber-bia** [comment]
In French we use the word "cépage" for the English "cultivar",  synonymous with "cultivated variety" (which is not the same as a botanical variety)

In some cases, the cultivar is the same as the variety, as in the case of Syrah, but in other cases, as in the case of Grenache, the grape cultivar has several varieties: Grenache noir, Grenache gris and Grenache blanc.

Just a remark here: the color of the wine is not directly related to the color of the grapes since maceration of the skin did not always occur ...so yes it will be complicated to fit for purpose wine style (Red Wine, White Wine, Rose Wine).

**Comment by DianeKEPR** [comment]
Good point.
Looks like people do technically and incorrectly mix the terms cultivar and variety when dealing with wine 

This description from:  https://en.wikipedia.org/wiki/List_of_grape_varieties
The term grape variety refers to cultivars rather than actual botanical varieties according to the International Code of Nomenclature for Cultivated Plants, because they are propagated by cuttings and may have unstable reproductive properties. However, the term variety has become so entrenched in viticulture that any change of usage to the term cultivar is unlikely.

https://en.wikipedia.org/wiki/Grenache_blanc
Grenache blanc (also known as garnatxa blanca in Catalonia) is a variety of white wine grape that is related to the red grape Grenache. It is mostly found in Rhône wine blends and in northeast Spain. 

Then there is also the case of:
https://en.wikipedia.org/wiki/Tempranillo_blanco
Tempranillo blanco is a white Spanish wine grape variety that is grown in the Rioja Denominación de Origen (DOC). It is a mutation of the red Tempranillo grape variety that is planted in Rioja. The white grape variety was discovered in a Tempranillo vineyard in the Rioja region by a wine grower in the late 20th century. In 2007, the Consejo Regulador of Rioja officially sanctioned the use of Tempranillo blanco in the DOC wines of Rioja.

This about Grenache Gris
https://www.wine-searcher.com/grape-1384-grenache-gris
Grenache Gris is a pinkish-grey mutation of the red Grenache grape and is grown to a limited extent in the south of France. Little research has been conducted into the history of Grenache Gris and the variety remains in relative obscurity.
Like its family members Grenache and Grenache Blanc, Grenache Gris is vigorous, resistant to drought and prefers warm, dry environments.
Varietal wines made from Grenache Gris are something of a rarity and a winemaker's indulgence, since there is little commercial gain to be made from producing them.

https://www.masterclass.com/articles/learn-about-grenache-wine-grape-history-characteristics-and-different-grenache-wines#what-climate-is-best-for-growing-grenache
Like many ancient grape varieties, grenache has mutated a number of times. These mutations are genetically identical to the parent but are different in appearance.
Grenache noir is the original and most common grenache, whose thin skins result in a medium-ruby colored wine. It is made into red wines as well as rosé styles.
Grenache blanc, a white mutation of grenache, is grown in northeast Spain as well as France’s Rhône Valley. White Priorat wines made from grenache blanc and other white grapes are gaining popularity, and grenache blanc has long been an important blending grape in rich southern Rhône whites.
Grenache gris, a mutation named after the grayish-pink color of its skin, is less well-known. 

https://winefolly.com/grapes/pinot-noir/
Pinot Noir is one of the few red grapes that’s commonly made into red, rosé, white, and sparkling wine.
DNA analysis has revealed that Pinot Noir, Pinot Gris, and Pinot Blanc are simply mutations of the same grape
https://winefolly.com/deep-dive/whats-the-difference-between-pinot-gris-and-pinot-grigio/
You probably already knew that Pinot Gris and Pinot Grigio are the same wine.
What’s surprising is that this grayish-purple grape is also a mutation of Pinot Noir. 

https://en.wikipedia.org/wiki/Cabernet_Sauvignon
Despite its prominence in the industry, the grape is a relatively new variety, the product of a chance crossing between Cabernet Franc and Sauvignon blanc during the 17th century in southwestern France. Its popularity is often attributed to its ease of cultivation—the grapes have thick skins and the vines are hardy and naturally low yielding, budding late to avoid frost and resistant to viticultural hazards such as rot and insects—and to its consistent presentation of structure and flavours which express the typical character ("typicity") of the variety. 
Cabernet blanc is a crossing of Cabernet Sauvignon and an unknown hybrid grape variety that was discovered in Switzerland in the late 20th century.

**Proposed 1:**
Under each species of grape plant, create instances (not subClasses) using each cultivar/variety name.   Question:  where does the concept of variety/cultivar belong.  Is there something under each species, a subClass of each species called for example,  vitus vinifera grape plant variety  ( and here is attached the data property for grape skin color, that is inherited by each variety(cultivar)
**Proposed 2:**
For varieties that result in a mutation from the most common grape variety, set up these mutation varieties as separate instances, not subClasses as a result of mutations.  My point of view: reflecting the mutation relationship would be fit for purpose for a genome grape domain, but adds complexity for a Food perspective of grapes.  Same structure for varieties as a result of cross-cultivation with 2 varieties.
**Proposed 3:**
For each grape variety, use the short name for the grape, not appending the phrase "....grape plant".   (this point of view should be considered along with the upcoming issue) Wine Definition.
**Proposed 4:**  (this one could be modeled several ways, each with pros and cons of alternative approaches)   For each grape variety instance, have the information/value of the color classification (white grape or red grape) associated in some structure to assign the grape color classification to each grape variety/cultivar instance.  Red Grape is any grape with skin color pigment (Red, Black, Rose Red, Purple Red),  White Grape is any grape skin without color pigment  (green / yellow)

This proposal would look like:
grape plant
----vitus vinifera  (syn: common grape plant, european grape plant)   ?add a line below for vitus vinifera grape plant variety
--------chardonnay   hasGrapeSkinColor    (? White,  or  White Grape)
--------pinot noir       hasGrapeSkinColor    (? Red,  or Red Grape)
--------pinot gris (syn: pinot grigio)  hasGrapeSkinColor  (? Red,  or Red Grape) the skin is pink, but usually made into white wine with a light grey hue
--------pinot blanc    hasGrapeSkinColor    (? White,  or White Grape)
--------tempranillo      hasGrapeSkinColor    (? Red,  or Red Grape)
--------tempranillo blanco   hasGrapeSkinColor    (? White,  or White Grape)
--------grenache  (syn: grenache noir, garnacha)   hasGrapeSkinColor    (? Red,  or Red Grape)
--------grenache blanc   hasGrapeSkinColor    (? White,  or  White Grape)
--------grenache gris   hasGrapeSkinColor    (? Red,  or Red Grape)  the skin is pink, but usually made into white wine with a light grey hue
--------cabernet sauvignon    hasGrapeSkinColor    (? Red,  or Red Grape)
--------cabernet franc    hasGrapeSkinColor    (? Red,  or Red Grape)
--------sauvignon blanc   hasGrapeSkinColor    (? White,  or  White Grape)
--------cabernet blanc    hasGrapeSkinColor    (? White,  or  White Grape)

**Comment by maweber-bia** [comment]
Red Grape is any grape with skin color pigment (Red, Black, Rose Red, Purple Red), White Grape is any grape skin without color pigment (green / yellow) --> I agree, this should be reflected in the definition

I don't know how to deal with "variety/cultivar" but I think your proposed 4 "For each grape variety instance, have the information/value of the color classification (white grape or red grape) associated in some structure to assign the grape color classification to each grape variety/cultivar instance." looks like a good solution, since then you do not need to group the cultivar/variety by color, which could be tricky.

**Comment by DianeKEPR** [comment]
Proposed:  establish the existence of grape skin.
a grape plant   derives:
grape stem
grape leaf
grape fruit   --- Grape (whole, raw)

grape fruit   derives:
grape skin
grape seed
grape pulp

grape skin  has skin color   with domain values:   Red, White

Model grape plant cultivars  and grape cultivars  similar to the FoodOn example for:  Fuji Apple
for the purpose of wine common language:  grape cultivar  has a synonym of   grape variety
each grape cultivar instance  has skin color    with values either Red  or  White

Modeling the various color hues that can be associated with grape skin color  Red or White  ; would be considered out of scope

**Comment by ddooley** [comment]
So I suggest locating these ideas within the framework of existing taxonomy and plant anatomy ontologies and their concepts, as well as reusing RO/anatomical relationships.
(Taxonomy has received a boost from genomics, so now its much easier to order our hierarchy of organisms by genetic origin. ) 
- NCBITaxon describes species and subspecies-level organisms. I'm no expert on these, but I can say that we simply create a subclass of http://purl.obolibrary.org/obo/NCBITaxon_29760 Vitis vinifera if we don't already have one for a given wine grape, and perhaps we position it within any grape variety it is genetically descended from (this may be another point of debate). The "grape plant" suffix helps to distinguish when references are being made to a plant as a whole vs. its fruit.  Whether something has a "cultivar" or a "variety" suffix is a bit arbitrary but its ok to add/follow industry conventions there in the term name since scientific names are less frequently available at that level. Also, in the OBO Foundry realist view, instances are reserved for things in the real world - a particular mutated plant say.  If there is a class of plant just like it (having the same DNA, carrying the same mutation), we'd always define a class for that, not an instance.

- For every plant X, we can have a X berry term defined explicitly (aka precomposed). You will see this structure established in robot management file for wine plants and grapes: https://docs.google.com/spreadsheets/d/1jcZtqoN-C75d4OCPUomtp5xNbqjLweoooSVqpK3XHSY/edit#gid=0

- the Plant ontology has terms for berry fruit, skin and seed.  A grape is a berry fruit from Vitis vinifera; FoodOn has a definition: "'berry fruit' and ('derives from' some 'grape plant')". I see this "grape plant" flies in at Vitis genus level rather than Vitis vinifera, so that should be discussed/verified.

- FoodOn already has terms for black, red and white grapes, although they are not attached to "black, red or white grape plant classes, which I think do need to be created.

- FoodOn has terms for fruit skin colour, so we would want to reuse those: http://purl.obolibrary.org/obo/FOODON_03530278.  These haven't been enhanced with PATO colour vocabulary yet though, so that is another to-do.

On this last point I'm going to pontificate a bit in preparation for the upcoming [Wine Ontology Workshop at BioHackathon](https://github.com/elixir-europe/bioHackathon-projects-2021/tree/main/projects/16), because it will be a great discussion topic there.  

- Saying "cabernet sauvignon hasGrapeSkinColor (? Red, or Red Grape)" illustrates a preference for named relations like "hasGrapeSkinColor" that actually hide semantics, which I am weary of, but willing to accept as a kind of compiled output product of a purer semantic representation.  Nothing tells a computer that hasGrapeSkinColor means that a grape plant's grape's skin has a given colour - its more of a database shortcut. A purer representation of this is " 'cabernet sauvignon' 'has part' some ('grape (whole)' and 'has part' some ([exocarp](http://purl.obolibrary.org/obo/PO_0009085) and 'has quality' some [red](http://purl.obolibrary.org/obo/PATO_0000322) ))".  In this way other problem domains can know exactly what they are dealing with that might be pertinent.  This exposes the threshold between "deep" and "lite" ontologies!

**Comment by ddooley** [comment]
About "These mutations are genetically identical to the parent but are different in appearance" ... this can't be true unless mechanisms external to the genome are controlling expression of apearance?

**Comment by maweber-bia** [comment]
> 
> 
> About "These mutations are genetically identical to the parent but are different in appearance" ... this can't be true unless mechanisms external to the genome are controlling expression of apearance?

Yes, there are external mechanisms such as terroir (soil), climate, etc. which have influence/impact on expression of appearance  (cf nature versus nurture !) It is a big scientific/agronomic question to deal with.

**Comment by ddooley** [comment]
Great to have you starting the day and on the board!  But I'd rephrase "These mutations are genetically identical to the parent" - A mutation is implicitly with respect to a parent, so the two can't be the same?  One could just say a varietal has a different appearance based on terroir, climate, etc... ?  Maybe my understanding of "mutation" is just limited to bacteria / viruses.

**Comment by maweber-bia** [comment]
I was jumping into the discussion without reading throughout, but now referring to Diane's proposed 2  :  "For varieties that result in a mutation from the most common grape variety, set up these mutation varieties as separate instances, not subClasses as a result of mutations. My point of view: reflecting the mutation relationship would be fit for purpose for a genome grape domain, but adds complexity for a Food perspective of grapes. Same structure for varieties as a result of cross-cultivation with 2 varieties.) " 


--> my answer would be :  yes, a mutation is implicit with respect to a parent, so the two are not  the same based on genetics

--> and yes : One could just say a varietal has a different appearance based on terroir, climate, etc... 

I hope this will help !!
see you

**Comment by DianeKEPR** [comment]
Per a comment from Damion: yes, do recommend modeling similar to Pome Fruit (and associated structure) as in the Foodon example.
Thinking that GrapeSkinColor has values Red and White.  Would indicate has pigment or does not have pigment.   Black might be considered as a color Red.  Might be GrapeSkinColorClassification.  Red or White would be skin color associated to each variety/cultivar.   Black can be used, dark purple grapes rarely used to make GrapeWine.   This would be taken care of with assigning a Wine Style - Red, White, Rose -- there would not be a Wine Style called Black Wine.
Recommend having variety/cultivar as one concept --I think Variety as the primary label, with Cultivar as a synonym --  difficult to gather data to support the distinction, less complex in the context of GrapeWine -- leans more to a binary dataproperty on GrapeWine to be: is Varietal?
A dataproperty such as Color Hue could be used if want to distinguish light green, yellow-green, dusky pink, red, black ( this is dark purple red), purple etc.
A mutation on a variety with red grapes to now produce white grapes (absence of pigment) creates a new Variety (Tempranillo and White Tempranillo) -- do not need a relationship of which was a mutation of the other (unless needed for a different context purpose)
Concepts such as Terroir should apply related to Geography / Wine Regions.  Terroir can influence taste of a given Variety, but in of itself does not create a different variety.  Not sure Terroir is needed for Fit for Purpose of this ontology.

**Comment by DianeKEPR** [comment]
https://www.sciencedaily.com/releases/2007/03/070306101301.htm
White grapes are a result of a mutation that occurred in various cultivars that produce red grapes.   Other source: it is thought that historically, all grapes were red (all wild grapes are red) and later the white grapes came to be.

**Comment by DianeKEPR** [comment]
The below description probably does not conform to the taxon design in FoodOn, but will let Damion work that into the ontology.
A common wording in Wine is the notion of Varietal wines.   A wine from a single variety (other factors) is considered to a varietal.
Proposing to have a single level for a notion of Grape Variety that is syn with Cultivar.  In the taxon view of plants, not sure how to depict a class for Grape Variety.
So, in that perspective, I am modeling the concept of Grape Plant Species -- with instances for each species:  Vitis vinifera, Vitis labrusca, Vitis aestivalis  ....plus others.
A separate class for Grape Plant  Variety  -- with instances for Chardonnay, Pinot Noir, Tempranillo, White Tempranillo,   ... plus many other.
Create object properties to relate the instance of species with the variety
-----   Vitis vinifera    expressed as   Chardonnay     and so on with each  species to variety/cultivar
Grape color to use the current FOODON  color classes:   White,  Red ,  Black
----- for wine, the factor is "with pigment"   or  "without pigment";  so do not need the color Hue
----- can regard that Red Wine can be made from  Red Grapes  or   Black Grapes  (we would not ask anyone for Black Wine)   interesting that the pulp of Black Grapes is light green, just that the pigment color bleeds from the skin immediately upon crushing to get your purple grape juice.
seems there are several techniques to model valid values of data properties in either sub-classes or instances 
----  so in simple format  (Damion can change to show berry through exocarp)   
----------   there would be a class  " Grape color"    with instances of   Red, White, Black
--------have an object property to express  that  Cultivars   have an instance of    Grape Color
--------  with instance relationships such as    Chardonnay   has grape color     White
Pinot Noir   has grape color   Red,    ....  and so on.

**Comment by DianeKEPR** [comment]
Another way to approach (creating alternatives)
Have a backbone ontology approach where
    Genus   hasSpecies     Species
    Species  hasVariety      Variety    (Syn:  Cultivar)   
    subClasses (from some place, but not a taxonomy of plant terms)  -->   Genus,  Species,  Variety
    Vitis   isa   Genus     (syn:  GrapeVine)    -->  has stems, berries, exocarp (skin), leaves   etc.
    Vitis Vinifera     isSpeciesOf     Vitis
    Vitis Lambrusca   isSpeciesOf  Vitis      plus about 80+ other Species
    Chardonnay   isa  Variety
    Pinot Noir  isa   Variety
    Chardonnay isVarietyOf  Vitis Vinifera
    PinotNoir  isVarietyOf   Vitis Vinifera

might take a path where  (for purpose of the Wine domain)
    Vitis   has   GrapeBerries
    GrapeBerries   has    GrapeColor
    GrapeColor   isa   Class
     Instances of GrapeColor   -->   RedGrape, WhiteGrape, BlackGrape

In WineStyle   RedWine  can be made with  RedGrapes  or BlackGrapes    for wine the condition is, grape skin has pigment  (Red Grape),  grape skin lacks pigment  (White Grape)
Black Grapes are grapes with a dark purple where the pigment bleeds almost immediately into the juice upon crushing.    Pinot Noir  most often is a vary dark purple but it takes more time for the pigment to bleed into the juice -- and is considered a Red Grape.    BlackGrape to be used in creating Red Wine, one would not ask for a Black Wine.  Black grapes have a light color pulp.
     Chardonnay   hasGrapeColor    WhiteGrape
     PinotNoir      hasGrapeColor      RedGrape

**Comment by DianeKEPR** [comment]
White wine from red grapes

White Pinot Noir is richer than many white wines because it’s made with red wine grapes. It has flavors of baked apple and pear, with zesty notes of honey, orange and ginger. Depending on how it’s made, the color can range from a pale white gold to a deep saffron yellow.  

White Pinot Noir is made like a white wine. It is juice fermenting in the absence of skins and a very different fermentation than red wine fermentation. [Anne Amie Vineyards’] is barrel fermented, like Chardonnay. Others are tank fermented in stainless steel. It is a much cooler/slower fermentation than red wine fermentation.  
Our style uses fully-ripe Pinot Noir. A small portion of free-run juice is liberated by the press and once that is done the grapes get sorted a second time and made into Pinot Noir. Some wineries do opt to use underripe Pinot Noir for their wines.

The red colour of wine comes from the grape skins, not the pulp, i.e. the juicy bit of the grape. In theory any red grape can make a white wine, as it just depends how long the skins, with their red colour compounds, were left in contact with the clear juice.

If you peel any red grape you will see this for yourself. The skins will again your fingers red, but the juicy pulp inside is not red. (Only a tiny minority of red grape varieties have red flesh).

Champagne and sparkling wines made in the ‘traditional method’ aka Champagne method are produced using a process called Direct Pressing:

Whole bunches of grapes are very gently pressed, to burst the skins, releasing the juice, but the juice is run off before it has time to be stained by the skins.

In red wine making you do the opposite. You crush and leave juice and skins in contact to macerate, extracting as much colour and tannin as desired.

For Rosé winemaking you do something in between.

NB the traditional Champagne terminology of ‘Blanc de Blanc’; white from white (typically Chardonnay) or ‘Blanc de Noir’; white from black (typically Pinot Noir and Pinot Meunier.
Champagne uses the clear juice from Pinot Noir grapes to make their sparkling wines. This has been true for centuries. It is often blended with Chardonnay and the lesser-known Pinot Meunier for this purpose. When Champagne uses only Pinot Noir juice to make champagne, it is designated as Blanc de Noir (White from Black)
